core/state/database.go

```diff
@@ -121,14 +138,56 @@ func NewDatabaseWithConfig(db ethdb.Database, config *trie.Config) Database {
 	}
 }
 
+func NewDatabaseWithConfigAndCache(db ethdb.Database, config *trie.Config) Database {
+	csc, _ := lru.New(codeSizeCacheSize)
```

Comment:
should we swallow the errors here

Reply:
![image](https://user-images.githubusercontent.com/7310198/126598627-f283424d-de6b-48ca-967c-4ffe3fbeb899.png)
Because only one error is possible, and all the size here is positive, so just ignore the error